allensdk/core/mouse_connectivity_cache.py

```diff
@@ -417,7 +417,7 @@ def get_experiment_structure_unionizes(self, experiment_id,
                                                 pre=col_rn,
                                                 post=filter_fn,
                                                 writer=lambda p, x : pd.DataFrame(x).to_csv(p),
-                                                reader=pd.DataFrame.from_csv)
+                                                reader=lambda x: pd.read_csv(x, index_col=0, parse_dates=True))
```

Comment:
Did you check to make sure that the file saved here puts the correct column first when written?

cc @NileGraddis

Reply:
I'll add a test that covers it. I was going by the documentation for to_csv (https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.to_csv.html) which indicates that by default it is writing an index label column.

In general I'm planning to add more tests like the one I added for get_ephys_features that actually round trip the files, as I noticed all the tests we have seem to mock out the file writing.